tests/minimal.wcon

```diff
@@ -0,0 +1 @@
+{"tracker-commons":true, "units":{}, "data":[]}
```

Comment:
I read the specification as being silent on which properties are required, only that the `"units"` object itself is required.

So, I've amended the `WCON_format.md` to be more explicit:

From

> `"units"` is required and must be single-valued.

to

> `"units"` is required and must be single-valued.  Units must be specified at least for `"t"`, `"x"`, and `"y"`.

I've also changed all my tests and examples to reflect this.